classes/webservice/WebserviceRequest.php

```diff
@@ -526,7 +558,7 @@ public function fetch($key, $method, $url, $params, $bad_class_name, $inputXml =
             $this->depth = isset($this->urlFragments['depth']) ? (int) $this->urlFragments['depth'] : $this->depth;
 
             try {
-                // Method below set a particular fonction to use on the price field for products entity
+                // Method below sets a particular function to use on the price field for products entity
                 // @see WebserviceRequest::getPriceForProduct() method
                 // @see WebserviceOutputBuilder::setSpecificField() method
                 //$this->objOutput->setSpecificField($this, 'getPriceForProduct', 'price', 'products');
```

Comment:
May be remove this commented line.

Reply:
I didn't touch this part so I thought it might be useful for contextual or historical information when it will be refactored.

Reply:
:ok_hand: